withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -134,21 +199,47 @@ DEFINE
  BNOPRINTDEBUGINFODAYS     = NUMBER
  BNODEBUGMODEOFF           = 0
  BNODEBUGMODEON            = 1
- BNOINDENTMAX              = 9
- BNOLOCALLOOP              = NUMBER
 END [DEFINE]
 
 SETUP
  Q=CTRLCHR(34)
+ QQ=CTRLCHR(34)
 
- INCLUDEPROGRAMINFO=TRUE
- BNOLASTMODDATE='10/23/23'
- BNOLASTMODTIME="17:30 MT"
+ BNOLASTMODDATE='11/01/23'
+ BNOLASTMODTIME="14:30 MT"
  BNOPROGRAMNAME="BANNO.CHECK.WITHDRAW.V1.POW"
  BNOCFGFILENAME="BANNO.CHECK.WITHDRAW.V1.CFG"
- BNOPROGRAMVERSION="1.1.1"
+ BNOPROGRAMVERSION="1.2.0"
  BNOPRINTDEBUGINFODAYS=90
 
+ JSONBOOLEAN(TRUE)="true"
+ JSONBOOLEAN(FALSE)="false"
+
+ BNOERRCOUNT=0
+ BNOERRMESSAGE=""
+ BNOERRDISPLAYMSGCOUNT=0
+
+ BNOERRCODE=BNOERRCODECFGPARSE          BNOERRMESSAGE="Error reading config file"
```

Comment:
Because this PO had a bunch of cases where a single ERRORCODE actually had multiple different text strings for the same BNOERRCODE.  So the initialization just sets the baseline text.  But then, in certain cases, the message gets overriden with more specific text.

Reply:
OK